### ISSUE TITLE
Why does the image named val_mIoU so strange?

### ISSUE THREAD

**Issue by Buling-Knight** [NVlabs/SegFormer#108]
My train loss has always decrease, but the val_mIoU decreased at the first time, then it began increase.
![val_mIoU](https://user-images.githubusercontent.com/91008145/194453445-1962823d-609b-456f-befd-5827f8aad37f.svg)

**Comment by Buling-Knight** [comment]
Should I increase the learning rate? Now it's 6e-5.
![learning_rate](https://user-images.githubusercontent.com/91008145/194456568-f2d6b46f-1360-4666-b6d5-4c82deebce10.svg)

**Comment by Buling-Knight** [comment]
Should I increase the learning rate ? Now it's 5e-6.
![learning_rate](https://user-images.githubusercontent.com/91008145/194456642-f9b36327-a230-4817-bd67-e74a83675aff.svg)
